articles/tutorials/create-a-sample-app/02-create-a-petclinic-csharp/create-a-petclinic-csharp.md

```diff
@@ -135,74 +135,109 @@ Create a new Package in which [DTO](https://en.wikipedia.org/wiki/Data_transfer_
 
 <p><video style="max-width: 100%" muted="true" loop="true" autoplay="true" src="videos/create-services-dtos-package.mp4"></video></p>
 
-Right-click on the `DTOs` package and create a new `DTO` with a name of `OwnerDTO`.
+Create the `OwnerDTO`:
 
-Also create also the following:
+- Right-click on the `DTOs` package and click the `New DTO` option.
+- Give it the name `OwnerDTO`.
+
+Also create also the following additional DTOs:
 
 - `PetDTO`
 - `PetVisitDTO`
+- `OwnerCreateDTO`
+
+Right-click the `OwnerCreateDTO` element and click the `Add Field` option to add the following fields:
```

Comment:
This was further down in the tutorial, inside in the section of creating the service, which felt out of place. So I've moved it here. Additionally it was just an image which means that someone using a screen reader wouldn't know what needs to be mapped.